### ISSUE TITLE
can conservation area box be automatically populated?

### ISSUE THREAD

**Issue by polly64** [colouring-london/colouring-core#13]
@matkoniecz can conservation area box be automatically populated easily now and if not can we say 'to be automatically populated'
<img width="339" alt="Screenshot 2023-01-15 at 19 14 39" src="https://user-images.githubusercontent.com/42236514/212562230-90b1fa45-0cfa-4860-9a4c-3170de0352fb.png">

**Comment by matkoniecz** [comment]
Yes, it could be filled by intersecting building areas and CA areas (one thing to figure out is licensing and whether it matters)

**Comment by polly64** [comment]
@matkoniecz is the message re listing in the box uindicating this? if so can we say ' In CA. Please provide CA link and also add 'CA ' at the end of hyperlink to'Conservation areas i.e. 'Conservation Areas (CA)' in the question
